build.sh

```diff
@@ -53,28 +51,26 @@ pushd .
 
 cd Example_DynamicFramework
 pod install
-pod update
-fastlane match adhoc --force_for_new_devices
-fastlane gym --scheme PointziDemo --export_method "ad-hoc" --output_directory "$BUILD_OUTPUTS" --output_name "PZDynamic.ipa" --clean true
+fastlane beta
 
 popd
 
 # ---------------------- upload to hockeyapp ---------------------
 
-/usr/local/bin/puck -submit=auto -download=true -notes="$(git log -1)" \
-                    -notes_type=markdown \
-                    -source_path=$(pwd) \
-                    -repository_url=https://github.com/StreetHawkSDK/ios \
-                    -api_token=$HOCKEYAPP_TOKEN \
-                    -app_id=$HOCKEYAPP_APPID_DYNAMIC \
-                    $BUILD_OUTPUTS/PZDynamic.ipa
-/usr/local/bin/puck -submit=auto -download=true -notes="$(git log -1)" \
-                    -notes_type=markdown \
-                    -source_path=$(pwd) \
-                    -repository_url=https://github.com/StreetHawkSDK/ios \
-                    -api_token=$HOCKEYAPP_TOKEN \
-                    -app_id=$HOCKEYAPP_APPID_STATIC \
-                    $BUILD_OUTPUTS/PZStatic.ipa
+#/usr/local/bin/puck -submit=auto -download=true -notes="$(git log -1)" \
```

Comment:
why is this commented ?

Reply:
it is replaced by a method hockey() in "fastlane beta"

Reply:
do you want me to delete it, in stead of comment out?

Reply:
remove it . and are you using the env variables ?